LoopKit/InsulinKit/PersistedPumpEvent.swift

```diff
@@ -28,6 +28,8 @@ public struct PersistedPumpEvent {
     public let type: PumpEventType?
     /// Whether the pump event is marked mutable
     public let isMutable: Bool
+    /// Whether the event was automatic or manually triggered. nil indicates this status is unknown.
+    public let automatic: Bool?
```

Comment:
[nit] I think we're trying to use `isBlah` or `hasBlah`, etc. for `Bool`s (e.g. `isMutable` above).  Consider `isAutomatic`?